ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestionProperties.java

```diff
@@ -1,5 +1,9 @@
 package com.microsoft.azure.kusto.ingest;
 
+import org.codehaus.jackson.map.ObjectMapper;
```

Comment:
Currently this is available through another dependency but should be included in the POM explicitly to avoid problems in the future.

Reply:
it is used in many other places in the library

Reply:
Right, but if it will be removed from there then you won't have it here and you will get the error only after "install" maven goal. It should be explicitly declared in each project it is being used at (probably we missed it before)

Reply:
its a